model/pipeline/src/main/proto/standard_window_fns.proto

```diff
@@ -29,26 +29,41 @@ option go_package = "pipeline_v1";
 option java_package = "org.apache.beam.model.pipeline.v1";
 option java_outer_classname = "StandardWindowFns";
 
+import "beam_runner_api.proto";
 import "google/protobuf/duration.proto";
 import "google/protobuf/timestamp.proto";
 
-// beam:windowfn:global_windows:v0.1
-// empty payload
+message StandardWindowFnUrns {
+  enum Enum {
```

Comment:
This is an interesting example. No WindowFn is required as a primitive. They are all just "well-known". There is no particular reason they should be in the same list, except the SDK *today* ships with them.

What about putting these in an enum inside their payload?

```
message FixedWindows {
   enum Urn {
     FIXED_WINDOWS = 0 [(beam_urn) = "beam:windowfn:fixed_windows:v0.1"]
   }
   google.protobuf.Duration size = 1;
   google.protobuf.Timestamp offset = 2;	   google.protobuf.Timestamp offset = 2;
}
```

(the `Payload` suffix has always been extraneous; I regret introducing it)

Reply:
We may require GlobalWindows (e.g. when we need to pass elements and don't care about the windowing). 

Putting these URNs as part of the payload is an interesting idea.